de/repositories.md

```
@@ -39,3 +39,62 @@ Um eine einfache Nutzung unserer Software zu ermöglichen, wollen wir möglichst
+* __Weniger Daten__ als bei Monorepos: Das Klonen und Nachvollziehen von Änderungen (z. B. mit `git blame`) geht schneller.
+* Fehler im __main branch__ betreffen nur einen Microservice, nicht das gesamte Produkt oder den gesamten Service.
+
+Multirepos eignen sich vor allem für eigenständige Services und insbesondere für Libraries, die in anderen Projekten genutzt werden sollen.
+
+### Monorepo
+
+> Spricht man von einem Monorepo-Ansatz, bedeutet dies im Prinzip, dass es nur ein einziges Repository im gesamten Unternehmen gibt. In diesem Repository sind  alle Projekte des Unternehmens enthalten. Dazu gehören sowohl aktive als auch inaktive Projekte.
+> Der bekannteste Fall eines Monorepos im produktiven Einsatz ist Google, das ein  eigenentwickeltes Sourcecode-Management-Tool gebaut hat, um damit genau ein  einziges Repository zu verwalten. Auch Meta Platforms, das Unternehmen hinter  Facebook und Instagram, setzt im Wesentlichen auf ein Monorepo, bei dem die Git-kompatible Eigenentwicklung [Sapling](https://engineering.fb.com/2022/11/15/open-source/sapling-source-control-scalable/) zum Einsatz kommt, die Ende 2022 als Open  Source veröffentlicht wurde.  
+> So ein Monorepo ist verdammt riesig und enthält die Historie von (fast) allen Projekten der Firma.
+> [^devops]
+
```

markdownlint-cli2 (0.23.1)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)